content/pt-br/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -0,0 +1,182 @@
+---
+title: EndpointSlices
+content_type: concept
+weight: 45
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.21" state="stable" >}}
+
+_EndpointSlices_ entregam uma maneira simples de rastrear endpoints de rede através de um cluster Kubernetes. Eles oferecem uma alternativa mais escalável e extensa aos Endpoints.
+<!-- body -->
+
+## Motivação
+
+A Endpoints API oferece uma maneira simples e direta de rastremento de endpoints de rede no Kubernetes. Infelizmente à medida que os clusters Kubernetes e os {{< glossary_tooltip text="Serviços" term_id="service" >}} vem crescido para lidar e mandar mais tráfego para mais Pods do backend, limitações da API original se tornaram mais visíveis.
+Mais notavelmente, estas limitações incluíam desafios com escalar para um númerio maior de endpoints de rede.
```

Comment:
"Notably" can also have the meaning of especially or in particular. How about changing this to "Em particular, estas limitações incluíam desafios de escalonamento para um número maior..."?